scripts/sshzobelisk

```diff
@@ -0,0 +1,40 @@
+set -euo pipefail
+
+usage()
+{
+    echo "USAGE: $0 <USERNAME ON ZOBELISK> <USER ID ON ZOBELSIK>"
+    exit 1
+}
+
+if [ $# -ne 2 ]; then
+    usage
+fi
+
+LOCAL_USER=$USER
+SERVER_USERNAME=$1
+SERVER_UID=$2
+LOCAL_REPO=$(realpath $(dirname $BASH_SOURCE)/../)
+ZOBELISK_IP=192.168.37.176
+IP=`hostname -I`
+IP=`echo $IP | head -n1 | awk '{print $1;}'` # `echo $IP | sed 's/ *$//g'`
+echo $IP
```

Comment:
This looks like some debug leftovers, unclear why you define $IP over two lines

Reply:
this is parsing the IP's returned by hostname -I

Reply:
its alot simpler to define in this way

Reply:
This is beating a dead horse, but how is:
```
IP=`echo $IP | head -n1 | awk '{print $1;}'` | head -n1 | awk '{print $1;}'
```
"alot simpler"
than
```
IP=`hostname -I | head -n1 | awk '{print $1;}'` | head -n1 | awk '{print $1;}'
```

Reply:
Oh no you right, I thought youre meant something diffrent